### PR TITLE
Guide for ejabberd

### PR DESCRIPTION
**Pull request by fm0de**
Guide for ejabberd including Erlang installation. Could possibly split in two guides.

### CODE REVIEW

File: source/guide_ejabberd.rst [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+If it's not in state RUNNING, check your configuration.
+
+Finishing installation
+======================
+
+Register your administrator user:
+
+.. code-block:: console
+
+   [isabell@stardust ~]$ ejabberdctl register admin isabell.example <password>
```
Comment (clerie) [inline source/guide_ejabberd.rst]: inconsistent domain, you've used isabell.uber.space in line 394

File: source/guide_ejabberd.rst [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+    config.status: creating vars.config
+    config.status: creating src/ejabberd.app.src
+    [isabell@stardust ejabberd-19.05]$
+
+Install it:
+::
+
+    [isabell@stardust ejabberd-19.05]$ make install
+
+Make the controll script available:
```
Comment (clerie) [inline source/guide_ejabberd.rst]: typo in control

**Comment by nichtmax** [comment]
We'll provide erlang on all systems in one of the next Uberspace 7 updates, we'll keep you posted and then review your guide.

**Comment by nichtmax** [comment]
Update on Erlang: [close to rollout](https://github.com/Uberspace/manual/issues/86).

**Comment by jugendhacker** [comment]
I have two little thoughts about this:

1. It would be cool if you could also add how the SRV records need to be configured that this works without manually putting in the Port and Host (look at https://codimd.c3d2.de/s/ejabberd-uberspace#Last-missing-DNS-Records)

2. It would be cool to als say something about that mnesia (internal database) is not the best and that the user should switch to mysql or postgre if they want things like MAM work correctly

File: source/guide_ejabberd.rst [hunk 3]
```diff
@@ -0,0 +1,503 @@
...
+
+Provide location of your keys and certificates for TLS transport:
+
+.. note:: The list can be expanded with certfiles for each domain and subdomain.
+.. code-block:: ini
+ :emphasize-lines: 2,3
+
+  certfiles:
+    - "/home/isabell/etc/certificates/isabell.example.crt"
+    - "/home/isabell/etc/certificates/isabell.example.key"
```
Comment (jugendhacker) [inline source/guide_ejabberd.rst]: I think there should be somehow written that you also need to include the certs for the subdomains

**Comment by Schlumpf** [comment]
Erlang has arrived!

**Comment by clerie** [comment]
I've tested this guide before and had some issues.
Now, since Uberspace provides native Erlang support, I've updated my notes and published a guide on my blog.

https://blog.clerie.de/ejabberd-auf-uberspace-installieren/

Sadly, it's in german and I do not have the time to learn rst currently. But maybe some wants to translate it and bring it in a form suitable for Uberlab.

If you have any questions or having trouble reproducing this guide, please let me know.

**Comment by nichtmax** [comment]
I'm really sorry but this is too complex to support, there hasn't been _one_ of our team who got it working. [Shoot us a mail](mailto:hallo@uberspace.de), you'll get your reward anyway of course ☺️

**Pull request closed without merging**